tests/robot-tests/tests/admin/analyst/expired_invitation.robot

```diff
@@ -0,0 +1,42 @@
+*** Settings ***
+Library             ../../libs/admin_api.py
+Resource            ../../libs/admin-common.robot
+
+Suite Setup         user opens the browser    %{ADMIN_URL}
+Suite Teardown      user closes the browser
+Test Setup          fail test fast if required
+
+Force Tags          Admin    Local    Dev
+
+
+*** Test Cases ***
+Invite user to the service with an expired invite via the API
+    user adds user invite via api
+    ...    %{EXPIRED_INVITE_USER_EMAIL}
+    ...    Analyst
+    ...    2022-10-01 11:00:00
+
+Login with an expired invite and assert that the user is redirected to the expired invite page
+    user navigates to admin frontend
+    user waits until h1 is visible    Sign in    %{WAIT_MEDIUM}
+    user clicks element    id:signin-button
+    user logs in via identity provider
+    ...    %{EXPIRED_INVITE_USER_EMAIL}
+    ...    %{EXPIRED_INVITE_USER_PASSWORD}
+
+Check that the user is on the Expired Invite page
+    user waits until page contains title    Invitation expired
+    user waits until page contains    Your invitation to the service has expired.
+    user waits until page contains    explore.statistics@education.gov.uk
+
+Check that the expired invite is now removed
+    user navigates to admin frontend
+    user waits until h1 is visible    Sign in    %{WAIT_MEDIUM}
+    user clicks element    id:signin-button
+
+    ## Currently, users attempting to login with no existing user record or user invite
```

Comment:
Don't think double hashtags are needed here

Reply:
Right you are - just left single ones, thanks